src/containers/Sidebar/Sidebar.test.jsx

```diff
@@ -285,6 +286,38 @@ describe('<Sidebar />', () => {
     render(<SidebarWrapper store={store} />);
     const subscriptionManagementLink = screen.getByRole('link', { name: 'Subscription Management' }, { exact: false });
     expect(subscriptionManagementLink).toBeInTheDocument();
+    expect(subscriptionManagementLink).toHaveAttribute('href', '/test-enterprise-slug/admin/subscriptions');
+  });
+
+  it('renders correctly when enableReportingConfigScreen is false', () => {
+    const store = mockStore({
+      sidebar: {
+        ...initialState.sidebar,
+      },
+      portalConfiguration: {
+        enableReportingConfigScreen: false,
+      },
+    });
+    features.REPORTING_CONFIGURATIONS = true;
+    render(<SidebarWrapper store={store} />);
+    const enableReportingConfigScreenLink = screen.queryByRole('link', { name: 'Reporting Configurations' }, { exact: false });
+    expect(enableReportingConfigScreenLink).toBeNull();
+  });
+
+  it('renders correctly when enableReportingConfigScreen is enabled', async () => {
+    const store = mockStore({
+      sidebar: {
+        ...initialState.sidebar,
+      },
+      portalConfiguration: {
+        enableReportingConfigScreen: true,
+      },
+    });
+    features.REPORTING_CONFIGURATIONS = true;
+    render(<SidebarWrapper store={store} enableReportingConfigScreen />);
+    const enableReportingConfigScreenLink = screen.getByRole('link', { name: 'Reporting Configurations' }, { exact: false });
```

Comment:
Do we need `{ exact: false }` here?  TMK it isn't used in role queries (as opposed to text queries).